src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions_CreateEqualsMethod.cs

```diff
@@ -107,6 +107,13 @@ public static IMethodSymbol CreateEqualsMethod(this Compilation compilation, Imm
         {
             var statements = ArrayBuilder<SyntaxNode>.GetInstance();
 
+            // A ref like type can only return false from equals.
```

Comment:
add comment that doesn't explain what we're doing, but why we're doing it .

Reply:
Fixed